template_feed/Microsoft.DotNet.Test.ProjectTemplates.1.x/content/MSTest-VisualBasic/Company.TestProject1.vbproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' == ''">netcoreapp2.0</TargetFramework>
+    <TargetFramework Condition="'$(TargetFrameworkOverride)' != ''">TargetFrameworkOverride</TargetFramework>
+
+    <IsPackable Condition="'$(EnablePack)' == 'true'">true</IsPackable>

```

Comment:
We have this for C# https://github.com/dotnet/templating/blob/rel/vs2017/post-rtw/template_feed/Microsoft.DotNet.Test.ProjectTemplates.2.0/content/MSTest-CSharp/Company.TestProject1.csproj#L7-L8